src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -81,6 +71,15 @@ public Memo getMemo() {
     return mMemo;
   }
 
+  /**
+   * Get the Soroban data for the transaction
+   *
+   * @return SorobanTransactionData
+   */
+  public Optional<SorobanTransactionData> getSorobanData() {
+    return mSorobanData;
```

Comment:
`Optional` usage here is fine, can convert the nullable class variable with `Optional.of(mSorobanData)`, or skip it and just doc it as nullable

Reply:
Fixed in 37e0e19accdadde748a715af4e5dbf25a73ad40e